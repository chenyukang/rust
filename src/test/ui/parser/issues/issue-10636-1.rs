struct Obj { //~ NOTE unmatched opening delimiter begins here
    member: usize
)
//~^ ERROR mismatched closing delimiter
//~| NOTE mismatched closing delimiter

fn main() {}
