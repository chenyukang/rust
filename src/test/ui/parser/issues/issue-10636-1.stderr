error: unexpected closing delimiter: `)`
  --> $DIR/issue-10636-1.rs:4:1
   |
LL | )
   | ^ unexpected closing delimiter

error: this file contains an unclosed delimiter
  --> $DIR/issue-10636-1.rs:8:14
   |
LL | struct Obj {
   |            - unclosed delimiter
...
LL | fn main() {}
   |              ^

error: expected `,`, or `}`, found keyword `fn`
  --> $DIR/issue-10636-1.rs:3:18
   |
LL |     member: usize
   |                  ^ help: try adding a comma: `,`

error: functions are not allowed in struct definitions
  --> $DIR/issue-10636-1.rs:8:1
   |
LL | struct Obj {
   |        --- while parsing this struct
...
LL | fn main() {}
   | ^^^^^^^^^^^^
   |
   = help: unlike in C++, Java, and C#, functions are declared in `impl` blocks
   = help: see https://doc.rust-lang.org/book/ch05-03-method-syntax.html for more information

error: aborting due to 4 previous errors

